django/applications/catmaid/control/object.py

```diff
@@ -21,7 +21,7 @@ def get_annotation_graph(project_id=None):
         class_instance_b__class_column__class_name__in=["root", "group", "neuron", "skeleton"],
     ).select_related("class_instance_a", "class_instance_b", "relation",
         "class_instance_a__class_column", "class_instance_b__class_column")
-    g=nx.DiGraph()
+    g=nx.DiGraph() # XXX Will this just fail if the import of nx fails?
```

Comment:
worth thinking about - will remove comment before commit

Reply:
Yes and unfortunately we also don't give any warning about the import like we do with other optional imports. In fact, `networkx` isn't really optional given it is part of the main `requirements.txt` file and we should instead remove the `try/except` of the `nx` and `json_graph` imports.